### DISCUSSION TITLE
Remove filter icon with feedback if filters are in custom display mode

### DISCUSSION THREAD

**Discussion by sashberd** [KevinVandy/material-react-table#1314] [Q&A]
Hi
I set `columnFilterDisplayMode: 'custom',` and it hide the filter icon 
So I added my custom implementation for column Header property with filter icon
However, if you apply filter from my custom filter component that filter icon is back with filter notification popover
![image](https://github.com/user-attachments/assets/080ea2e9-f733-4250-a288-e6b75a2dc4a3)
IMO, if we use custom filter display mode this one should not be shown at all, since also filter icon not shown
How to get rid of this icon?

**Comment by Duoquote** [comment]
Did you find a solution? I am also tinkering with the same thing.

**Comment by sashberd** [comment]
@Duoquote I made some workaround
1. I added to the table props:
```
icons: {
 FilterAltIcon: () => <div id="fake-filter-icon-id" />,
},
```

2.  Than in my global css I added:
```
 button:has(#fake-filter-icon-id) {
                display: none;
              }
```

So, till the author add fix, this workaround works for me

**Reply by Duoquote** [reply]
My workaround was not using table's filtering at all, as I was already going to remotely fetch results, I decided to store my own filtering rules elsewhere.
